### DISCUSSION TITLE
Could not resolve dependency

### DISCUSSION THREAD

**Discussion by darkdim** [larowlan/vite-plugin-twig-drupal#8] [General]
![Screenshot 2023-12-20 175958](https://github.com/larowlan/vite-plugin-twig-drupal/assets/9481910/087fd9f2-0da5-423b-a79a-c23bde8dbcd2)

**Comment by larowlan** [comment]
Are you using storybook 7 with the vite builder?

**Comment by darkdim** [comment]
> Are you using storybook 7 with the vite builder?

I used this manual for installation. Yes, the latest version is indicated there, i.e. the latest one is current.
I understand that the module is set to an older version, which is why the error occurs.
https://medium.com/@askibinski/drupal-10-single-directory-components-storybook-23d9ec59ecae
